rs/boundary_node/rate_limit_canister/interface.did

```diff
@@ -0,0 +1,89 @@
+type Version = nat64;   // Represents the config version number
+type Timestamp = nat64; // Represents timestamp in nanoseconds since the epoch (1970-01-01)
+type RuleId = text;     // Unique identifier for each rule
+
+
+// Input structure for defining a rule with mandatory fields within a config
+type InputRule = record {
+  id: RuleId;        // Unique identifier for the rule
+  rule_raw: blob;    // Raw rule data (in binary format), expected to be a valid json object
+  description: text; // Textual description of the rule
+};
+
+// Output structure for rules
+// Optional fields rule_raw and description may remain hidden while the rule is under confidentiality restrictions
+type OutputRule = record {
+  id: RuleId;                        // Unique identifier for the rule
+  rule_raw: opt blob;                // Raw rule data (in binary format), expected to be a valid json object, none if the rule is currently confidential
+  description: opt text;             // Textual description of the rule, none if the rule is currently confidential
+  disclosed_at: opt Timestamp;       // Timestamp when the rule was disclosed, none if the rule is still confidential
+  disclosed_in_version: opt Version; // Version in which the rule was disclosed, none if the rule is still confidential
```

Comment:
One of the two is not needed. Version and Timestamp are the same information. I would drop the timestamp.

Reply:
This is a rather different information. I'd encourage to keep both.

Reply:
removed, as agreed